source/whats-new.txt

```diff
@@ -0,0 +1,41 @@
+==========
+What's New
+==========
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+Learn what's new in:
+
+* :ref:`Version 4.0 <version-4.0>`
+* :ref:`Version 3.6 <version-3.6>`
+
+.. _version-4.0:
+
+What's New in 4.0
+-----------------
+
+New features of the 4.0 Node.js driver release include:
```

Comment:
This feels incomplete. Is this meant to target the master branch and 4.0 or just the 3.6 branch?